src/auxiliary/t8code.jl

```diff
@@ -0,0 +1,465 @@
+"""
+    init_t8code()
+"""
+function init_t8code()
+  t8code_package_id = t8_get_package_id()
+  if t8code_package_id >= 0
+    return nothing
+  end
+
+  # Initialize `p4est` with log level ERROR to prevent a lot of output in AMR simulations
+  t8_init(SC_LP_ERROR)
+
+  return nothing
+end
+
+@inline t8_mpi_comm() = mpi_comm().val
+
+function trixi_t8_unref_forest(forest)
+  t8_forest_unref(Ref(forest))
+end
+
+function t8_free(ptr)
+  sc_free(t8_get_package_id(), ptr)
+end
+
+function finalize_t8code()
+  sc_finalize();
+  return nothing
+end
+
+function trixi_t8_count_interfaces(forest)
+  # /* Check that forest is a committed, that is valid and usable, forest. */
+  @T8_ASSERT (t8_forest_is_committed(forest) != 0);
+
+  # /* Get the number of local elements of forest. */
+  num_local_elements = t8_forest_get_local_num_elements(forest);
+  # /* Get the number of ghost elements of forest. */
+  num_ghost_elements = t8_forest_get_num_ghosts(forest);
+  # /* Get the number of trees that have elements of this process. */
+  num_local_trees = t8_forest_get_num_local_trees(forest);
+
+  current_index = 0
+
+  local_num_conform = 0
+  local_num_mortars = 0
+  local_num_boundry = 0
+
+  for itree = 0:num_local_trees-1
```

Comment:
```suggestion
  for itree in 0:(num_local_trees - 1)
```
This is the style we usually use in Trixi.jl. Please follow it (to make @sloede happy and the whole code easier to read)

Reply:
I am fully convinced that it will make everyone happier and live longer 😎

Reply:
Oh no ... my P***on lout came to the surface. :D

Reply:
Still open